tutorial/loadable-components.md

```diff
@@ -44,6 +98,7 @@ export async function renderLayout(res, params) {
         ...
         topbar: renderLoadableReduxComponent(Header, params.store, extractor),
         navbar: renderLoadableReduxComponent(NavBar, params.store, extractor),
+        footer: renderLoadableReduxComponent(Footer, params.store, extractor),
```

Comment:
I think we need to inform to hydrate the necessary component also (if it is rendering from server-side, event wont work we need to hydrate it )